spec/support/skippable.rb

```diff
@@ -0,0 +1,18 @@
+RSpec.configure do |config|
+  config.before(:all) do
+    $skip = false
```

Comment:
Tried using `@@skip` here, but can't use class variables in rspec. I couldn't figure out any other way to share state across examples.

Reply:
Probably not Luke have to make a Skip class or module with top level var. I’m fine either way.